From 427efe0a95ed3d0e2e912a1cb91d31605d465e74 Mon Sep 17 00:00:00 2001
From: Dan Walkes <danwalkes@trellis-logic.com>
Date: Wed, 31 Oct 2018 14:46:43 -0600
Subject: [PATCH] Update environment defaults for tegra

Remove definitions for CONFIG_ENV_OFFSET, CONFIG_ENV_OFFSET_REDUND,
and partition definition, allowing these to be configured in yocto mender
framework instead.
---
 include/config_defaults.h | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/include/config_defaults.h b/include/config_defaults.h
index 2b503f8..3b0e251 100644
--- a/include/config_defaults.h
+++ b/include/config_defaults.h
@@ -22,10 +22,12 @@
 
 #endif
 #define CONFIG_ENV_SIZE 0x20000
-#define CONFIG_ENV_OFFSET 0x400000
-#define CONFIG_ENV_OFFSET_REDUND 0x800000
+/* Let mender define these */
+/*#define CONFIG_ENV_OFFSET 0x400000 */
+/*#define CONFIG_ENV_OFFSET_REDUND 0x800000 */
 #define CONFIG_SYS_MMC_ENV_DEV 0
-#define CONFIG_SYS_MMC_ENV_PART 0
+/* Don't use this definition, per mender requirements */
+/*#define CONFIG_SYS_MMC_ENV_PART 0 */
 #define CONFIG_ENV_IS_IN_MMC
 #define CONFIG_BOOTCOUNT_LIMIT
 #define CONFIG_BOOTCOUNT_ENV
-- 
2.7.4

